File: problemas/iniciante/1930/1930.go
package main

import (
    "fmt"
)

func main() {
    var t1, t2, t3, t4 int

    fmt.Scanf("%d %d %d %d", &t1, &t2, &t3, &t4)
    soma := t1 + t2 + t3 + t4 - 3
    fmt.Printf("%d\n", soma)
}

File: problemas/iniciante/1930/1930.exs
defmodule Modulo do
  def leitura(entrada, n) do
    if n > -1 do
      numero = entrada |> Enum.at(n) |> String.to_integer()
      numero + leitura(entrada, n - 1)
    else
      0
    end
  end
end

entrada = IO.gets("") |> String.trim |> String.split(" ")
soma = Modulo.leitura(entrada, 3)
IO.puts (soma - 3)